sky/shell/platform/ios/FlutterView.mm

```diff
@@ -30,4 +39,25 @@ - (BOOL)enableInputClicksWhenVisible {
   return YES;
 }
 
+/* The container itself is not accessible but contains accessible elements. */
+- (BOOL)isAccessibilityElement {
+  return NO;
+}
+
+- (NSArray*)accessibleElements {
+  return _accessibleElements;
+}
+
+- (NSInteger)accessibilityElementCount {
```

Comment:
The accessibleElements accessor won't be needed - I don't think the others are implied though.